pilot/pkg/networking/core/v1alpha3/listener_inbound.go

```diff
@@ -129,6 +129,10 @@ func (cc inboundChainConfig) ToFilterChainMatch(opt FilterChainMatchOptions) *li
 		} else {
 			match.PrefixRanges = IPv6PassthroughCIDR
 		}
+
+		if features.EnableDualStack && cc.clusterName == util.InboundPassthroughCluster {
```

Comment:
Did you change the name before.?
features.EnableDualStack check is redundant, InboundPassthroughCluster can be only for dual stack

Reply:
Yes,  exactly speaking, I merged `InboundPassthroughBindIpv4` and `InboundPassthroughBindIpv6` into a single cluster `InboundPassthroughCluster` (Changed from `InboundPassthroughClusterForDS` into `InboundPassthroughCluster` in previous PR), so yes, the cluster name has been changed.

Right, features.EnableDualStack check is redundant, I will remove it.

Reply:
I am thinking maybe we donot need dual stack cluster, keep two separate passthrough clusters (ipv4/ ipv6). And let the filterchain match select which cluster to use.

Reply:
Yeah, that's also an option for this. If do not merge, we can save more effort.

Reply:
Hi @hzxuzhonghu, I submitted again, no merged for original destination passthrough clusters.